### PR TITLE
added battery status

### PR DESCRIPTION
**Pull request by midzer**
I wanted to implement a rudimentary battery status check.
Until now there are only two possible states: "good" and "low".

Maybe i figure out how much voltage the battery delivers if being fully charged.
Then one can add a third state like "full"

### CODE REVIEW

File: campapp/main.c [hunk 1]
```diff
@@ -39,8 +39,13 @@ void infoscreen(){
     lcdPrintln("DOWN: DFU mode");
     lcdNl();
     lcdPrintln("UP:   MSC");
-    lcdPrintln("  (to copy files");
-    lcdPrintln("     to badge)");
+    lcdPrintln("(copy files to badge)");
```
Comment (penguineer) [inline campapp/main.c:42]: On my device this does not fit in one line. Why has the line break been removed?
(Battery status still would fit)

**Comment by penguineer** [comment]
Tested it on my device and the code worked. :)

See above comment.
Also, I'd rather see a loading percentage or sth alike – I'm a bit unsure here. From the usability standpoint, Low/Good/Full are better than just showing a number. As an engineer, I'd like some more fine-grained feedback.

**Comment by Fensterbank** [comment]
@penguineer: There is now a graphical battery l0adable (with current value and a battery graphic),  which was merged on https://github.com/rad1o/f1rmware/pull/64.
I think this PR can be closed.

**Comment by midzer** [comment]
yes, @Fensterbank. Your linked PR offers the same features with neat graphics, too

**Pull request closed without merging**